### PR TITLE
OCT-1685: Implement "Recalculating score" modal: client -- UI only

### PR DESCRIPTION
**Pull request by jmikolajczyk**
## Definition of Done

1. [ ] Acceptance criteria are met.
2. [ ] PR is manually tested before the merge by developer(s).
    - [ ] Happy path is manually checked.
3. [ ] PR is manually tested by QA when their assistance is required (1).
    - [ ] Octant Areas & Test Cases are checked for impact and updated if required (2).
4. [ ] Unit tests are added unless there is a reason to omit them.
5. [ ] Automated tests are added when required.
6. [ ] The code is merged.
7. [ ] Tech documentation is added / updated, reviewed and approved (including mandatory approval by a code owner, should such exist for changed files).
    - [ ] BE: Swagger documentation is updated.
8. [ ] When required by QA:
    - [ ] Deployed to the relevant environment.
    - [ ] Passed system tests.

---

(1) Developer(s) in coordination with QA decide whether it's required. For small tickets introducing small changes QA assistance is most probably not required.

(2) [Octant Areas & Test Cases](https://docs.google.com/spreadsheets/d/1cRe6dxuKJV3a4ZskAwWEPvrFkQm6rEfyUCYwLTYw_Cc).

### CODE REVIEW

File: client/src/components/Settings/SettingsProgressPath/SettingsProgressPath.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import cx from 'classnames';
+import { motion } from 'framer-motion';
+import React, { FC, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import styles from './SettingsProgressPath.module.scss';
+
+const SettingsProgressPath: FC = () => {
```
Comment (aziolek) [inline client/src/components/Settings/SettingsProgressPath/SettingsProgressPath.tsx:8]: No children, no props, no need for an `FC`.
```suggestion
const SettingsProgressPath: (): ReactElement => {
```

File: client/src/components/Settings/ModalSettingsRecalculatingScore/ModalSettingsRecalculatingScore.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  const { t } = useTranslation('translation', { keyPrefix: 'views.settings' });
+
+  return (
+    <Modal
+      bodyClassName={styles.modalBody}
+      dataTest="ModalSettingsRecalculatingScore"
+      header={t('recalculatingScore')}
+      isOpen={modalProps.isOpen}
+      onClosePanel={() => {
+        modalProps.onClosePanel();
```
[the remark this answers is not in the document]
Comment (jmikolajczyk) [reply client/src/components/Settings/ModalSettingsRecalculatingScore/ModalSettingsRecalculatingScore.tsx:23]: Done

File: client/src/components/Settings/SettingsProgressPath/SettingsProgressPath.tsx (hunk 1 above)
Comment (jmikolajczyk) [reply client/src/components/Settings/SettingsProgressPath/SettingsProgressPath.tsx:8]: It has props in next PRs

File: client/src/components/Settings/SettingsProgressPath/SettingsProgressPath.module.scss [hunk 3]
```diff
@@ -0,0 +1,82 @@
+.root {
```
[the remark this answers is not in the document]
Comment (jmikolajczyk) [reply client/src/components/Settings/SettingsProgressPath/SettingsProgressPath.module.scss:1]: Fixed -> I didn't have defined formatter for .scss files

**Review by aziolek — APPROVED** [review]

**Pull request merged**